model/host/host.go

```diff
@@ -26,6 +26,7 @@ type Host struct {
 	Tag      string        `bson:"tag" json:"tag"`
 	Distro   distro.Distro `bson:"distro" json:"distro"`
 	Provider string        `bson:"host_type" json:"host_type"`
+	IPv6	 string		   `bson:"ipv6_address" json:"ipv6_address"`
```

Comment:
Since this is its primary address, i.e., the thing we're going to return to the user, and we will return no other IP, should this instead be IP? (which might allow us in the future to use this for any kind of IP address, if a host provider doesn't provide a DNS name)

Reply:
Yeah might as well!